rust-runtime/aws-smithy-types/src/futures_stream_adapter.rs

```diff
@@ -17,6 +17,7 @@ use std::task::{Context, Poll};
 /// new-type to enable the trait when it is required.
 ///
 /// This is meant to be used by codegen code, and users should not need to use it directly.
+#[derive(Debug)]
```

Comment:
can these be kept in aws-smithy-http? It's only used in unstable situations, right?

Reply:
It's also used by `ByteStream::into_async_read`.